src/components/AudioTrack/Waveform.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     ref="el"
-    class="waveform relative bg-dark-charcoal-04 overflow-x-hidden"
+    class="waveform relative bg-dark-charcoal-04 overflow-x-hidden focus:outline-none"
```

Comment:
Will this disable the extra outline on high contrast mode on Windows? 🤔 I think we shouldn't necessarily fight the natural focus outline when it exists.

Reply:
> The outline-none utility is implemented using a transparent outline under the hood to ensure elements are still visibly focused to Windows high contrast mode users.

No, thankfully! https://tailwindcss.com/docs/outline

Reply:
Right on!